### ISSUE TITLE
Insanely slow now

### ISSUE THREAD

**Issue by 05v** [BeepIsla/fake-stattrak#30]
Hey, I was wondering why the repeat feature was removed, as it now takes ages to increment

**Comment by BeepIsla** [comment]
It was removed because it seems to have confused a lot of people, the program now simply does 10K increments every 50 milliseconds

**Comment by 05v** [comment]
Do you have a functioning feature which is faster? I am trying to hit the kill limit on my stattrack knife so that it dissapears!

**Comment by 05v** [comment]
I know its possible to do because I did this in 2020

**Comment by BeepIsla** [comment]
Yeah if your kills reach too high the counter will simply disappear for some reason.

For your problem I could add back the "repeat" option but leave it optional so people get less confused about it. I chose 10K every 50ms because from a little bit of testing I've done I couldn't go any higher or my weapon simply wouldn't get any extra kills tracked at all.

**Comment by 05v** [comment]
Its because the bit intiger limit is hit, that's what im aiming for here with my knife - just looking for a quick fix because the same knife non stattrack is $500 more, this is simply better

is there a working version already? I tried a commit prior to the removal but it just gives me a shit ton of errors

**Comment by BeepIsla** [comment]
If a previous commit doe snot work then no, I would have to add it back

**Comment by 05v** [comment]
Okay, how long do u think that would be?

**Comment by BeepIsla** [comment]
Not long, few hours at most unless I run into issues

**Comment by 05v** [comment]
Okay forsure, drop me your ETH or BTC addy I'll send a tip for the time :)

**Comment by BeepIsla** [comment]
Valve definitely seems to have changed something, 10 minutes for roughly 20K added to my stattrak weapon while I wanted 500K? I remember it being a lot better

**Comment by 05v** [comment]
![image](https://user-images.githubusercontent.com/64431087/227295128-40940291-f845-4e37-bbae-98687f46a232.png)
It would take 2 years to hit it otherwise

**Comment by 05v** [comment]
Oof, nah thats not worth it at all haha, 2 years is a pain - no other way?

**Comment by 05v** [comment]
I cry, thank u for checking in, seems like Valve patched it then :(

**Comment by 05v** [comment]
Yeah I see, there's no way to fuck around with like applying a stat track swap tool and changing the value in that request?

**Comment by BeepIsla** [comment]
The stattrak swap tool only takes two item IDs as input, the backend figures out the amount on its own

**Comment by 05v** [comment]
Ah fuck, there's nothing else I can think of myself

**Comment by 05v** [comment]
Unless there is a way to have multiple sessions play at the same time and farm

**Comment by BeepIsla** [comment]
In theory you can be authenticated to multiple gameservers at once but the backend ratelimit is likely just based on SteamID so that probably wouldn't help

**Comment by 05v** [comment]
Well fuck
